frontend/context/StakingContractDetailsProvider.tsx

```diff
@@ -122,10 +132,10 @@ const useStakingContractDetailsByStakingProgram = ({
       return Promise.allSettled(promises).then((results) => {
         const [stakingContractDetails, serviceStakingDetails] = results;
         return {
-          ...(stakingContractDetails.status === 'fulfilled'
+          ...(stakingContractDetails?.status === 'fulfilled'
             ? (stakingContractDetails.value as StakingContractDetails)
             : {}),
-          ...(serviceStakingDetails.status === 'fulfilled'
+          ...(serviceStakingDetails?.status === 'fulfilled'
```

Comment:
this was causing the flickering for "start agent & stake" button, basically was failing here